Documents/conversions/convert_sqlite.md

```diff
@@ -0,0 +1,145 @@
+# Converting an SQLite Table to other forms
+
+## From SQLite to PostgreSQL
+### Inputs
+* path to sqlite file
+* table name
+* PostgreSQL schema name
+* PostgreSQL table name (defaults to sqlite tablename)
+
+### PostgreSQL prerequisites
+
+The database is called `desc_dc2_drp`.  In order to make changes to the database you must have access to the account `desc_dc2_drp_admin`.  By default PostgreSQL applications like the command-line tool psql will look for account information  in the file `.pgpass` in your home directory, readable (and writable) only by you.  Then you can invoke psql as follows:
+```
+$ psql desc_dc2_drp desc_dc2_drp_admin
+psql (10.15, server 9.6.19)
+Type "help" for help.
+
+desc_dc2_drp=>
+```
+For more information about psql see [https://www.postgresql.org/docs/9.6/app-psql.html](https://www.postgresql.org/docs/9.6/app-psql.html).
+
+### Creating the PostgreSQL schema
+
+From the SQLite table schema make an equivalent .sql file, say called `my_create.sql`, to create the PostgreSQL table. You can see an SQLite schema from within the SQLite command line program by typing
+```
+sqlite> .schema
+```
+Here is the correspondence between SQLite type names and PostgreSQL type names for those types occuring in truth catalogs:
+
+| SQLite         | PostgreSQL       |
+|----------------|------------------|
+| TEXT           | TEXT             |
+| INTEGER or INT | INTEGER          |
+| BIGINT         | BIGINT           |
+| FLOAT          | REAL             |
+| DOUBLE         | DOUBLE PRECISION |
+
+If the original table has ra,dec columns, add a column of type `earth` (a typical name for such a column is `coord` or something containing that string) to the PostgreSQL schema. All of this could perhaps be automated but if so, review output.
+
+Run it from psql:
```

Comment:
By "run it" does it mean to create a schema in PostgreSQL according to `my_create.sql`? It's not immediately clear to me.

Reply:
That line `\i my_create.sql;`  has the same effect as executing each of the statements in my_create.sql.   That file will certainly have a `create table` statement.   It likely will also have a line like
`create schema if not exists some-schema-name;` (unfortunately involving a different meaning for the word "schema"; that should be explained)
I don't think it's appropriate to turn this readme into an sql tutorial, but I could include an example of an actual file used to create a table.